script/generate-mime.rb

```diff
@@ -97,7 +98,7 @@ def get_matches(parent)
 puts "  # :nodoc:"
 puts "  EXTENSIONS = {"
 extensions.keys.sort.each do |key|
-  puts "    '#{key}' => '#{extensions[key]}',"
+  puts "    '#{key}' => ['#{extensions[key].join('\',\'')}'],"
 end
```

Comment:
I think we should not support other formats. It is ok to break this since it is private api.